commands/cluster_commands/install

```diff
@@ -52,53 +53,52 @@ echo "selected target platform: ${TARGET_PLATFORM}"
 # if CIVO, then pre-set the configuration params
 if [ "${TARGET_PLATFORM}" = $CIVO ]; then
 
+  # first check if civo is installed or not, if not? then install it
+  check_and_install_civo
 
-  if [ "${CIVO_CLUSTER_ID}" = "" ] || [ "${CIVO_MASTER_IP}" = "" ]; then
```

Comment:
@grapple-solutions removed this check, because it seemed insignificant at this point.